_docs/pages/instrumentation/common-use-cases.md

```diff
@@ -253,3 +253,62 @@ if request.get('debug'):
         tags={tags.SAMPLING_PRIORITY: 1}
     )
 ```
+
+### Tracing Message Bus Scenarios
+
+There are two message bus styles that should be handled, Message Queues and Publish/Subscribe (Topics).
+
+From a tracing perspective, the message bus style is not important, only that the span context associated with the producer is propagated to the zero or more consumers of the message. It is then the responsibility of the consumer(s) to create a span to encapsulate processing of the consumed message and establish a _FollowsFrom_ reference to the propagated span context.
+
+As with the RPC client example, a messaging producer is expected to start a new tracing Span before sending a message, and propagate the new Span along with that message. The following example shows how it can be done.
```

Comment:
subtle point, but I'd prefer `propagate the new Span's SpanContext along with that message.`

We should also clarify that the producer Span only lives as long as it takes to enqueue/publish the message; i.e., it does not wait for a dequeue-er/consumer before finishing the Span. (I think this is the only sane thing if someone really stops to think about it, but it's different enough from nested RPCs that it deserves to be stated explicitly)